public/app/plugins/datasource/prometheus/querybuilder/components/LabelFilterItem.tsx

```diff
@@ -0,0 +1,162 @@
+import debounce from 'debounce-promise';
```

Comment:
Ok, splitting this component for Loki and Prometheus: I'm ambivalent about this change, I don't like the duplication of the formatting functions, but this seemed like a good way not to break stuff. 
The actual diff from the original file is quite small, this is nearly identical to the [original](https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/prometheus/querybuilder/shared/LabelFilterItem.tsx), but this has a new function prop  (`getLabelValuesAutofillSuggestions`) that takes the current auto-complete string and returns potential matches. 

If the Loki maintainers decide that they also want to facilitate the auto-complete via the backend API, then we could go back to using a shared component?

Reply:
Created here: https://github.com/grafana/grafana/issues/58375